demo/Demos/ToolConsole.cs

```diff
@@ -91,7 +117,29 @@ public override async Task Run()
 		}
 	}
 
-	private static object[] GetTools() => [new GetWeatherTool(), new GetLatLonAsyncTool()];
+	private static void ListTools(object[] tools)
+	{
+		AnsiConsole.MarkupLine("\n[purple]Available tools:[/]");
+
+		foreach (var tool in tools)
+		{
+			if (tool is not OllamaSharp.Models.Chat.Tool chatTool)
+				break;
+
+			AnsiConsole.MarkupLineInterpolated($"{chatTool.Function?.Name ?? "Unknown"}\t\t[purple]{chatTool.Function?.Description}[/]");
+		}
+	}
+
+	private static async Task<object[]> GetTools(bool withMcpServers)
+	{
+		object[] tools = [new GetWeatherTool(), new GetLatLonAsyncTool()];
+
+		if (withMcpServers)
+			tools = tools.Union(await OllamaSharp.ModelContextProtocol.Tools.GetFromMcpServers("server_config.json")).ToArray();
```

Comment:
It seems that in this case, the file has to be located in the main repository folder. Maybe adding the executing assembly path´?

Reply:
Yep. Done